planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/src/node.cpp

```diff
@@ -133,6 +134,14 @@ ObstacleAvoidancePlanner::ObstacleAvoidancePlanner()
     declare_parameter("acceleration_for_non_deceleration_range", 1.0);
   traj_param_->max_dist_for_extending_end_point =
     declare_parameter("max_dist_for_extending_end_point", 5.0);
+  traj_param_->is_avoiding_unknown = declare_parameter("avoiding_object_type/unknown", true);
```

Comment:
A delimiter of nested parameter should be ".", so "avoiding_object_type/unknown" should be "avoiding_object_type.unknown".

Reply:
Thanks! I pushed the commit to modify the nested param.